types/react-native/test/fabric-component-sample.ts

```diff
@@ -0,0 +1,20 @@
+import codegenNativeComponent, { NativeComponentType } from 'react-native/Libraries/Utilities/codegenNativeComponent';
+import codegenNativeCommands from 'react-native/Libraries/Utilities/codegenNativeCommands';
+import type { ViewProps } from 'react-native';
+('use strict');
```

Comment:
ES strict mode is implied by using ES modules i.e. any `import` statement. This also not how you would invoke ES strict mode which is why the formatting is off. It would need to come before any other statement.

Reply:
Oh sorry I've copied that from [TurboModule test](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react-native/test/turbo-module-sample.ts#L2)